src/main/js/main_html5.js

```diff
@@ -336,8 +336,13 @@ require("../../../html5-common/js/utils/environment.js");
           _video.pause();
         } catch (ex) {
           // error because currentTime does not exist because stream hasn't been retrieved yet
-          console.log('VTC_OO: Failed to rewind video, probably ok; continuing');
+          OO.log('VTC_OO: Failed to rewind video, probably ok; continuing');
         }
+      } else if (!!rewind && OO.isEdge) {
+        // PBW-4555: Edge browser will always go back to time 0 on load.  Setting time to 0 here would
+        // cause the raw video element to enter seeking state.  If we call load while seeking on Edge,
+        // then seeking no longer works until the video stream url is changed.
+        currentTime = 0;
```

Comment:
I believe this code is easier to read :)

if&nbsp;(!!rewind)&nbsp;{
&nbsp;&nbsp;if&nbsp;(OO.isEdge)&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;//&nbsp;PBW-4555:&nbsp;Edge&nbsp;browser&nbsp;will&nbsp;always&nbsp;go&nbsp;back&nbsp;to&nbsp;time&nbsp;0&nbsp;on&nbsp;load.&nbsp;&nbsp;Setting&nbsp;time&nbsp;to&nbsp;0&nbsp;here&nbsp;would
&nbsp;&nbsp;&nbsp;&nbsp;//&nbsp;cause&nbsp;the&nbsp;raw&nbsp;video&nbsp;element&nbsp;to&nbsp;enter&nbsp;seeking&nbsp;state.&nbsp;&nbsp;If&nbsp;we&nbsp;call&nbsp;load&nbsp;while&nbsp;seeking&nbsp;on&nbsp;Edge,
&nbsp;&nbsp;&nbsp;&nbsp;//&nbsp;then&nbsp;seeking&nbsp;no&nbsp;longer&nbsp;works&nbsp;until&nbsp;the&nbsp;video&nbsp;stream&nbsp;url&nbsp;is&nbsp;changed.
&nbsp;&nbsp;&nbsp;&nbsp;currentTime&nbsp;=&nbsp;0;
&nbsp;&nbsp;}&nbsp;else&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;try&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;if&nbsp;(OO.isIos&nbsp;&&&nbsp;OO.iosMajorVersion&nbsp;==&nbsp;8)&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//&nbsp;On&nbsp;iOS,&nbsp;wait&nbsp;for&nbsp;durationChange&nbsp;before&nbsp;setting&nbsp;currenttime
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;$(_video).on("durationchange",&nbsp;_.bind(function()&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_video.currentTime&nbsp;=&nbsp;0;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;currentTime&nbsp;=&nbsp;0;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;},&nbsp;this));
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}&nbsp;else&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_video.currentTime&nbsp;=&nbsp;0;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;currentTime&nbsp;=&nbsp;0;
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_video.pause();
&nbsp;&nbsp;&nbsp;&nbsp;}&nbsp;catch&nbsp;(ex)&nbsp;{
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;//&nbsp;error&nbsp;because&nbsp;currentTime&nbsp;does&nbsp;not&nbsp;exist&nbsp;because&nbsp;stream&nbsp;hasn't&nbsp;been&nbsp;retrieved&nbsp;yet
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;OO.log('VTC_OO:&nbsp;Failed&nbsp;to&nbsp;rewind&nbsp;video,&nbsp;probably&nbsp;ok;&nbsp;continuing');
&nbsp;&nbsp;&nbsp;&nbsp;}
&nbsp;&nbsp;}
}